SpectraLogic.SpectraStorageBrokerClient/ResponseParsers/DeleteFileResponseParser.cs

```diff
@@ -16,23 +16,28 @@
 using log4net;
 using SpectraLogic.SpectraStorageBrokerClient.Model;
 using SpectraLogic.SpectraStorageBrokerClient.Runtime;
+using SpectraLogic.SpectraStorageBrokerClient.Utils;
+using System.Linq;
 using System.Net;
 
 namespace SpectraLogic.SpectraStorageBrokerClient.ResponseParsers
 {
-    internal class DeleteFilesResponseParser : IResponseParser<IJob>
+    internal class DeleteFileResponseParser : IResponseParser<Void>
```

Comment:
why does delete no longer return a value?

Reply:
the old code i had was just a guess on what stuff is going to look like, now that Ryan implemented it a saw that the delete call is not returning what i though it will.
make sense?